packages/components/package.json

```diff
@@ -19,6 +19,10 @@
       "import": "./dist/package/components/*",
       "types": "./dist/package/solid-components.d.ts"
     },
+    "./unversioned-utilities/*": {
```

Comment:
Info: Exports will be optimized and simplified when the architecture is optimized.